Rakefile

```diff
@@ -1,10 +1,38 @@
 #!/usr/bin/env rake
 
 require 'rake/testtask'
+require 'fileutils'
+
+TEST_TMP_DIR = 'test/tmp'
```

Comment:
The path `test/tmp` was chosen because it was one of the already ignored paths in `.gitignore`